src/FunctionApps/python/IntakePipeline/transform.py

```diff
@@ -1,2 +1,68 @@
-def transform_bundle(bundle: dict) -> None:
-    pass
+from smartystreets_python_sdk import us_street
+
+from phdi_transforms.basic import transform_name, transform_phone
+from phdi_transforms.geo import geocode
+
+
+def find_patient_resource(bundle: dict) -> dict:
+    """Grab a patient resource out of the bundle, and return a reference"""
+
+    patients = [
+        r
+        for r in bundle.get("entry")
+        if r.get("resource").get("resourceType") == "Patient"
+    ]
+
+    if patients:
+        return patients[0]
+
+
+def transform_bundle(client: us_street.Client, bundle: dict) -> dict:
+    """Standardize name and phone, geocode the address"""
+
+    resource = find_patient_resource(bundle)
+    if not resource:
+        return None
+
+    patient = resource.get("resource")
+
+    # Transform names
+    for name in patient.get("name", []):
+        if "family" in name:
+            name["family"] = transform_name(name["family"])
+
```

Comment:
Is family name only ever a single value and never a list?

Reply:
I've only ever seen it as a single value. Given name comes as a list, but I haven't seen that for family name. It's something we could do defensively if we wanted but I'm fine doing single value for now.

Reply:
Let's wait until we see it as a list somewhere, no need to get super defensive for the prototype